modules/tinymce/project.json

```diff
@@ -1,32 +1,21 @@
 {
   "name": "tinymce",
-  "namedInputs": {
-    "tinymce": [
-      "ts-globals",
-      "{projectRoot}/tsconfig.json",
-      "{projectRoot}/src/**/main/**/*.ts"
-    ]
-  },
+  "tags": [ "open-source", "editor" ],
   "targets": {
     "build": {
       "inputs": [
-        "tinymce",
-        "{projectRoot}/Gruntfile.js"
+        "tiny-product",
+        "^tiny-product"
+      ],
+      "dependsOn": [
+        "lint",
```

Comment:
Just trying to understand when and how the `lint` command runs. Will nx automatically run the `lint` command instead of it being part of the `prodBuild` grunt task?

Reply:
Ah, I wasn't thinking about TinyMCE building standalone. I'll make a second `build` target for standalone.